src/ape/api/transactions.py

```diff
@@ -294,6 +294,18 @@ def validate_txn_hash(cls, value):
     def call_tree(self) -> Optional[Any]:
         return None
 
+    @cached_property
+    def debug_logs(self) -> List[List[Any]]:
+        """Return any debug log data outputted by the transaction"""
+        return []
+
+    def print_debug_logs(self):
```

Comment:
To align with other similar methods on this class:

```suggestion
    def show_debug_logs(self):
```